.github/workflows/gh-actions-release.yml

```diff
@@ -0,0 +1,85 @@
+name: 'GH Actions - Release'
+
+on:
+  release:
+    types:
+      - created
+      - edited
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - FROM:     'ubuntu:focal'
+            COMPILER: 'gcc'
+            FLAGS:    '-DUSE_PYTHON_3=ON'
+          - FROM:     'ubuntu:focal'
+            COMPILER: 'clang'
```

Comment:
let's capture #400 in here too.

Reply:
Oops! Fixed. I don't know how I missed that earlier